techniques/css/C34.html

```diff
@@ -102,31 +98,31 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
+				<p class="note-title marker">注記</p>
+				<p>スティッキー領域は、キーボード利用者にとって不利になる可能性があるため、慎重に使用する必要があることに注意する。固定ヘッダーのあるページをタブで移動するキーボード利用者にとっての問題は、ページがスクロールを開始した後に、ページの上方にあるインタラクティブ要素に到達するために逆方向でタブ移動すると、多くの場合、フォーカスがスティッキーヘッダーの背後に移動した時点で見えなくなることである。そのため、利用者は、フォーカスを見るために上にスクロールしなければならないが、これは必ずしも意識して行っているわけではないだろう。同様に、目に見えるフォーカスはスティッキーフッターの後ろに消えてしまうので、利用者はフォーカスの位置を見るために下にスクロールする必要があり、これは非常に面倒である。</p>
 			</div>
 
-      <p>The basic approach for un-fixing sticky headers / footers is to:</p>
+      <p>固定されていないヘッダー/フッターの基本的な使い方は、次のとおりである:</p>
       <ol>
-        <li>Define the first sticky regions using media query <code class="language-css">min-height</code> properties, so they get fixed or un-fixed depending on the available space;</li>
+        <li>メディアクエリーの <code class="language-css">min-height</code> プロパティを使用して、最初のスティッキー領域を定義する。これにより、使用可能なスペースに応じて固定又は非固定にする。</li>
 
-        <li>Define other sticky regions using media query <code class="language-css">min-width</code> and <code class="language-css">max-height</code> properties for specific viewport sizes, so they get fixed or un-fixed depending on the available space, e.g. for tablets depending on the portrait or landscape position of the device.</li>
+        <li>特定のビューポートサイズのメディアクエリー <code class="language-css">min-width</code> 及び <code class="language-css">max-height</code> プロパティを使用して他のスティッキー領域を定義する。これにより、例えばタブレットの場合、デバイスの縦向き又は横向きの位置の使用可能なスペースに応じて固定又は非固定にする。</li>
       </ol>
 
     </section>
     <section id="examples">
-      <h2>Examples</h2>
+      <h2>事例</h2>
       <section class="example" id="example-1-un-fix-sticky-headers-/-footers-in-html-and-css">
-        <h3>Example 1: Un-fix sticky headers / footers in HTML and CSS</h3>
-        <p>The following example uses HTML and CSS to un-fix sticky headers / footers. The sticky regions get un-fixed as the height of the viewport is limited or the orientation is changed. When the <code class="language-css">min-height</code> property matches the viewport space defined via media queries, regions which are not sticky get fixed or vice versa. This particular example uses the CSS <code class="language-css">min-height</code>, <code class="language-css">max-height</code> and <code class="language-css">min-width</code> media query properties.</p>
+        <h3>事例 1: HTML 及び CSS でスティッキーヘッダー/フッターを固定しない</h3>
+        <p>次の例では、HTML 及び CSS を使用して、スティッキーヘッダー/フッターの固定を解除している。ビューポートの高さが制限される又は方向が変更されると、スティッキー領域は固定されなくなる。<code class="language-css">min-height</code> プロパティがメディアクエリーによって定義されたビューポートのスペースに一致する場合、スティッキーでない領域は固定され、その逆も同様となる。この例では、CSS の <code class="language-css">min-height</code>、<code class="language-css">max-height</code>、及び code class="language-css">min-width</code> メディアクエリープロパティーを使用する。</p>
```

Comment:
`<code>` タグが一部欠損していました。
```suggestion
        <p>次の例では、HTML 及び CSS を使用して、スティッキーヘッダー/フッターの固定を解除している。ビューポートの高さが制限される又は方向が変更されると、スティッキー領域は固定されなくなる。<code class="language-css">min-height</code> プロパティがメディアクエリーによって定義されたビューポートのスペースに一致する場合、スティッキーでない領域は固定され、その逆も同様となる。この例では、CSS の <code class="language-css">min-height</code>、<code class="language-css">max-height</code>、及び <code class="language-css">min-width</code> メディアクエリープロパティーを使用する。</p>
```